src/core/thread/mle.cpp

```diff
@@ -2420,6 +2421,53 @@ otError Mle::SendOrphanAnnounce(void)
     return error;
 }
 
+#if OPENTHREAD_CONFIG_MLE_LINK_METRICS_ENABLE
+otError Mle::SendLinkMetricsManagementResponse(const Ip6::Address &aDestination, otLinkMetricsStatus aStatus)
+{
+    otError  error = OT_ERROR_NONE;
+    Message *message;
+    Tlv      tlv;
+    ot::Tlv  statusSubTlv;
```

Comment:
It is necessary. Without `ot::`, `Tlv` would be regarded as `Mle::Tlv` whose `SetType` method takes an `Mle::Type as parameter. However, `kLinkMetricsStatus` is defined in `Mle` namespace and its type doesn't match `Mle::Type`. Removing `ot::` would cause compiling error.